### ISSUE TITLE
indent rule fails for structure in variable declaration

### ISSUE THREAD

**Issue by webOS101** [eslint/eslint#8829]
**Tell us about your environment**

* **ESLint Version:** 4.1.1
* **Node Version:** 6.8.0
* **npm Version:** 4.6.1 (because @5 poops on everything)

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

```
{
	"rules": {
		"indent": [1, "tab"]
	}
}
```

**What did you do? Please include the actual source code causing the issue.**

```js
var
	prop = {
		a: true
	},
	ri = 5;
```

**What did you expect to happen?**

No warnings

**What actually happened? Please include the actual, raw output from ESLint.**
```
  3:1  warning  Expected indentation of 3 tabs but found 2  indent
  4:1  warning  Expected indentation of 2 tabs but found 1  indent

✖ 2 problems (0 errors, 2 warnings)
  0 errors, 2 warnings potentially fixable with the `--fix` option.
```

If you reverse the two variable declarations, no warnings are given.

**Comment by platinumazure** [comment]
Hi @webOS101, thanks for the issue!

I think this might be a duplicate of #8785 and might be resolved by #8801 (merged into master but not yet released). If I'm right, this should be fixed in our minor release scheduled for 7 July (ish).

Maybe you could check against our master branch and see if the issue is still present? And/or feel free to close if you agree that this might be a duplicate of #8785. Thanks!

**Comment by webOS101** [comment]
Thanks for the super-fast response!  I did search issues and didn't see that one.  Oops.   I'll pull master and test to be sure.  If it's fixed, I'll close this.

**Comment by webOS101** [comment]
Looks to be fixed on my test case!  I can't test on my actual app as my locally linked custom config file is not found by the linked version of eslint.  Is that expected?  How do you work around that for testing?

**Comment by platinumazure** [comment]
@webOS101 Not sure I follow. Maybe you could stop by our [chat room](https://gitter.im/eslint/eslint) and see if anyone else can answer that question?

**Comment by webOS101** [comment]
Sure, I can try that.  To clarify (for posterity), I have globally installed all of my eslint plugins/configs.  I `npm link`ed `eslint`.  In my `package.json` I specify my main config file.  When I use the global `eslint` command (`eslint .`), it fails with a message indicating it cannot find `eslint-config-myconfig`.
